### DISCUSSION TITLE
导出功能的具体格式收集

### DISCUSSION THREAD

**Discussion by lmk123** [hcfyapp/crx-selection-translate#1765] [讨论]
最近我计划添加导出功能，但有一个问题，就是我不知道大家需要的具体的导出格式是什么样的。

比如，想导出成 txt 文件，那么这个 txt 文件的具体内容应该是什么样的？

目前有用户提供了一个导出成 excel 的具体内容，见 #1760，我会先添加这个导出格式。如果你期望添加别的格式，那么请在此帖子下面贴上示例文件

**Comment by Konjac-XZ** [comment]
赞成 Excel。

Excel 是对于主流用户而言通用度最高的格式，如果需要转换成其他格式的话完全可以自己动手。

**Reply by lmk123** [reply]
我计划提供多种格式以适应大部分软件，希望能尽可能免去用户“自己动手”的情况，但是我不知道哪些格式是用的比较多的 :joy:

**Reply by Konjac-XZ** [reply]
我描述一下我自己的需求：我是墨墨背单词的用户，这个 App 提供一个云词库的功能。将指定的文本上传到账户中，可以将文本中出现的单词加入自己的单词记忆规划。但是上传到账户是在网页端以纯文本的方式完成的，所以对我来说没区别。

**Comment by Konjac-XZ** [comment]
一个月过去了……我觉得我们可以加一个 XLSX，然后其他的再说……

**Comment by Konjac-XZ** [comment]
是不是大家都忘了这茬子事了，试图进行催更。
![image](https://github.com/hcfyapp/crx-selection-translate/assets/71483384/b166d788-002a-45de-95a0-fbeb43d49199)

**Reply by lmk123** [reply]
在做了在做了 :joy:
